### ISSUE TITLE
No unread indicator for messages received after disabling flight mode (going back online)

### ISSUE THREAD

**Issue by pavloburykh** [status-im/status-react#13444]
# Bug Report
Unread indicator does not appear for messages which where send to receiver during period of his being offline and received after going back online. 

**Steps:** 

1. Start 1-1 chat between Device 1 and Device 2 
2. Device 1 enables flight mode
3. Device 2 sends message to Device 1
4. Device 1 enters 1-1 chat and closes it while being offline
5. Device 1 disables flight mode and comes back online
6. On Device 1 pay attention if unread indicator appears for message received after disabling flight mode. 

**Actual result:** unread indicator does not appear. Message preview is presented. 

**Expected result:** unread indicator appears for message received after going back online. 

- Status version: nightly 
- Operating System: Android, iOS

https://user-images.githubusercontent.com/97245802/171643126-32fdac9b-cdac-430a-8710-db2eba6d2f19.mp4

[geth.log](https://github.com/status-im/status-react/files/8824461/geth.log)

**Comment by pavloburykh** [comment]
One more issue regarding unread indicator. Might be related to current issue although steps and actual behaviour differs little bit. Will leave it here for now. If this issue turns out to be not related then I will add separate bug for it. Just let me know.
 
### ISSUE 2 Messages in group chat are automatically read (unread indicator appears for second and then disappears) if received from recently unblocked user

**Steps:** 

1. User 1 and User 2 are in group chat
2. User 1 blocks User 2 and stops receiving messages from User 2 (as expected)
3. User 1 unblocks User 2 (but does not add User 2 in contact)
4. User 1 opens screen with chat list (do not enter existing group chat)
5. User 2 sends some messages in group chat
6. User 1 - pay attention at indicator of unread messages of given group chat

**Actual result:** unread indicator appears for second and then disappears. 

**Expected result:** unread indicator appears and remains until user reads messages in group chat (or marks all as read)


https://user-images.githubusercontent.com/97245802/171874822-11574c6d-b4a1-4e79-81f4-de81037b233f.mp4

[geth.log](https://github.com/status-im/status-react/files/8833037/geth.log)

**Comment by flexsurfer** [comment]
ISSUE1 : 
```
if chat.ReadMessagesAtClockValue > state.CurrentMessageState.WhisperTimestamp {
        receivedMessage.Seen = true
    }
```

**Comment by flexsurfer** [comment]
thanks @pavloburykh , can't replicate issue 2, strange that when you press back from group info, the home screen is opened instead chat screen, for me chat screen is opened, might be the issue, are you trying in the latest develop (nighlty) ?

**Comment by pavloburykh** [comment]
> thanks @pavloburykh , can't replicate issue 2, strange that when you press back from group info, the home screen is opened instead chat screen, for me chat screen is opened, might be the issue, are you trying in the latest develop (nighlty) ?

Hi @flexsurfer On given above video I press chat home button, not back. That's why I am redirected to chat screen. I am replicating the bug on latest nightly.

**Comment by flexsurfer** [comment]
hey @pavloburykh yes now i was able to reproduce, so the only thing you need is to use chat tab, could you create a separate issue pls

**Comment by pavloburykh** [comment]
> hey @pavloburykh yes now i was able to reproduce, so the only thing you need is to use chat tab, could you create a separate issue pls

Thanx @flexsurfer Done https://github.com/status-im/status-react/issues/13473
